<launch>
<arg name="Geta" value="15.0"/>
<arg name="namespace" value="tb3_0"/>

<param name="namespace_init_count" value="1"/>


  <node pkg="cqlite" type="global_frontier_detector" name="global__frontier_detector" output="screen">
  <param name="eta" value="$(arg Geta)"/>
  <param name="map_topic" value="$(arg namespace)/map"/>
  </node>
  
  <node pkg="cqlite" type="local__frontier_detector" name="local__frontier_detector" output="screen">
  <param name="eta" value="$(arg eta)"/>
  <param name="map_topic" value="$(arg namespace)/map"/>
  <param name="robot_frame" value="$(arg namespace)/base_link"/>
  </node>
 
  
  <node pkg="cqlite" type="frontier_filter.py" name="frontier_filter" output="screen">
  <param name="map_topic" value="$(arg namespace)/map"/>
  <param name="info_radius" value="1"/> 
  <param name="costmap_clearing_threshold" value="70"/> 
  <param name="goals_topic" value="/detected_points"/>
  <param name="namespace" value="tb3_"/> 
  <param name="n_robots" value="1"/>
  <param name="rate" value="100"/>
  </node>
  
  <node pkg="cqlite" type="cqlite_planner.py" name="cqlite_planner" output="screen">
  <param name="map_topic" value="$(arg namespace)/map"/>
  <param name="global_frame" value="map"/>
  <param name="info_radius" value="1"/> 
  <param name="info_multiplier" value="3.0"/> 
  <param name="hysteresis_radius" value="3.0"/> 
  <param name="hysteresis_gain" value="2.0"/> 
  <param name="frontiers_topic" value="/filtered_points"/> 
  <param name="n_robots" value="1"/>
  <param name="namespace" value="tb3_"/>
  <param name="delay_after_assignement" value="0.5"/>
  <param name="rate" value="100"/>
  </node>
    
</launch>
